box-content-sdk/src/main/java/com/box/androidsdk/content/auth/OAuthWebView.java

```diff
@@ -98,7 +98,7 @@ public void authenticate(final Uri.Builder authenticationUriBuilder){
     protected Uri.Builder buildUrl(String clientId, final String redirectUrl) {
         Uri.Builder builder = new Uri.Builder();
         builder.scheme("https");
-        builder.authority("app.box.com");
+        builder.authority("account.box.com");
```

Comment:
We might need to check if 3rd party developers are expected to use this end point.